lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction.js

```diff
@@ -0,0 +1,138 @@
+const clone = require('../../util/clone');
+const mixedNumber = require('./mixedNumber');
+
+const ChangeTypes = require('../../ChangeTypes');
+const Node = require('../../node');
+
+// Converts a mixed number to an improper fraction
+// e.g. 1 2/3 -> 5/3
+function convertMixedNumberToImproperFraction(node) {
+  if (!mixedNumber.isMixedNumber(node)) {
+    return Node.Status.noChange(node);
+  }
+
+  const substeps = [];
+  let newNode = clone(node);
+
+  // e.g. 1 2/3
+  const wholeNumber = mixedNumber.getWholeNumber(node); // 1
+  const numerator = mixedNumber.getNumerator(node); // 2
+  const denominator = mixedNumber.getDenominator(node); // 3
+  const isNegativeMixedNumber = mixedNumber.isNegativeMixedNumber(node);
+
+  // STEP 1: Convert to unsimplified improper fraction
+  // e.g. 1 2/3 -> ((1 * 3) + 2) / 3
```

Comment:
If Gen came up with this then I guess I'm okay to defer to her pedagogy, but I usually see mixed->improper done like this

5 2/3  -> 5 + 2/3 -> (5\*3)/3  + 2/3 -> (5\*3+2)/3 -> 5/3

I'd be confused about this first step as a student

but I just asked my friend who said she'd do it the way you're doing ^_^ so idk just my thoughts